site2/docs/tiered-storage-aws.md

```diff
@@ -47,14 +47,13 @@ This example uses Pulsar 2.5.1.
 
     **Output**
 
-    As shown from the output, Pulsar uses [Apache jclouds](https://jclouds.apache.org) to support [AWS S3](https://aws.amazon.com/s3/) and [GCS](https://cloud.google.com/storage/) for long term storage. 
```

Comment:
Why do you change this file?

Reply:
It does not display correctly.

Reply:
But this is not related with this change?

Reply:
@jiazhai this change is not related to logo, just fix issues together.